tests/core/process.c

```diff
@@ -713,11 +741,14 @@ store_export_address_if_mach_msg (const GumExportDetails * details,
 static gpointer
 sleeping_dummy (gpointer data)
 {
-  volatile gboolean * done = (gboolean *) data;
+  struct _TestThreadSyncData * sync_data = (struct _TestThreadSyncData *)data;
+  volatile gboolean * done = sync_data->done;
 
-  while (!(*done))
+  g_mutex_lock (&sync_data->thread_is_started_mutex);
+  g_cond_signal (&sync_data->thread_is_started);
+  g_mutex_unlock (&sync_data->thread_is_started_mutex);
+  while (!(* done))
```

Comment:
There should not be a space between `*` and `done`. Also a blank line before `while` would visually separate the two things that this function does.